rules/autoconsent/finsweet.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "finsweet",
+    "vendorUrl": "https://finsweet.com/components/cookie-consent",
+    "cosmetic": false,
+    "prehideSelectors": ["[fs-consent-element=banner]"],
+    "detectCmp": [
+        {
+            "exists": "[fs-consent-element=banner]"
+        }
+    ],
+    "detectPopup": [
+        {
+            "visible": "[fs-consent-element=banner] [fs-consent-element=allow]"
+        }
+    ],
+    "optIn": [
+        { "wait": 500 },
+        {
+            "waitForThenClick": "[fs-consent-element=banner] [fs-consent-element=allow]"
+        }
+    ],
+    "optOut": [
+        { "wait": 500 },
```

Comment:
Generic wait at the start of a rule seems like a bad pattern to me... is there something more specific we can wait for here?

Reply:
I tried to tweak the rule a bit, but the JS handlers are still slow to register sometimes, and it's not always visible from the DOM